src/main/java/org/fcrepo/importexport/common/FcrepoConstants.java

```diff
@@ -58,6 +58,7 @@
     public static final String REPOSITORY_NAMESPACE = "http://fedora.info/definitions/v4/repository#";
     public static final Resource INBOUND_REFERENCES = createResource(REPOSITORY_NAMESPACE + "InboundReferences");
     public static final Resource PAIRTREE = createResource(REPOSITORY_NAMESPACE + "Pairtree");
+    public static final Resource REPOSITORY_ROOT = createResource(REPOSITORY_NAMESPACE + "RepositoryRoot");
```

Comment:
Sorry, I forgot to include it in the non-test method where it was referenced (isRepositoryRoot), it is used now.